Algorithm.Python/QuandlFuturesDataAlgorithm.py

```diff
@@ -0,0 +1,53 @@
+﻿# QUANTCONNECT.COM - Democratizing Finance, Empowering Individuals.
+# Lean Algorithmic Trading Engine v2.0. Copyright 2014 QuantConnect Corporation.
+# 
+# Licensed under the Apache License, Version 2.0 (the "License"); 
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
+# 
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from clr import AddReference
+AddReference("System")
+AddReference("QuantConnect.Algorithm")
+AddReference("QuantConnect.Common")
+
+from System import *
+from QuantConnect import *
+from QuantConnect.Algorithm import *
+from QuantConnect.Python import PythonQuandl
+from datetime import datetime, timedelta
+
+class QCUQuandlFutures(QCAlgorithm):
+    '''QuantConnect University: Futures Example 
+    QuantConnect allows importing generic data sources! This example demonstrates 
+    importing a futures data from the popular open data source Quandl.
+    QuantConnect has a special deal with Quandl giving you access to Stevens Continuous Futurs (SCF) for free.
+    If you'd like to download SCF for local backtesting, you can download it through Quandl.com.'''
+
+    def Initialize(self):
+        ''' Initialize the data and resolution you require for your strategy '''
+        self.SetStartDate(2000, 1, 1)
+        self.SetEndDate(datetime.now().date() - timedelta(1))
+        self.SetCash(25000);
+
+        self.crude = "SCF/CME_CL1_ON"
```

Comment:
Please generally add more comments... e.g. "symbol corresponding to the quandl code."